server/index.js

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin / env node
+/*
```

Comment:
Nitpick: I find it nicer when there is an empty line between that first line and the license below.